pyvista/_vtk.py

```diff
@@ -566,6 +572,15 @@ def __init__(self):  # pragma: no cover
 
             raise VTKVersionError('vtkHDFReader requires VTK v9 or newer')
 
+    class vtkCGNSReader:  # type: ignore
```

Comment:
Do we even need this class here?  We can use a `lazy_vtkCGNSReader` function that raises the error.  I think this is causing one of the CI failures.

Reply:
Fixed in c2b36158ae864cebe208b2307ce30e3e3b1818fb